### PR TITLE
fix(bigquery): BQ batch size

### PR DESCRIPTION
**Pull request by bbernays**

### CODE REVIEW

**Review by disq — APPROVED** [review]

**Review by yevgenypats — APPROVED** [review]

File: plugins/destination/bigquery/client/client.go [hunk 1]
```diff
@@ -37,7 +38,7 @@ func New(ctx context.Context, logger zerolog.Logger, destSpec specs.Destination)
 	}
 
 	c.pluginSpec = spec
-
+	c.batchSize = spec.BatchSize
```
Comment (candiduslynx) [inline plugins/destination/bigquery/client/client.go:41]: where is it used?

Comment (candiduslynx) [reply plugins/destination/bigquery/client/client.go:41]: specifically, as thi is a new fields, where is it passed to the write itself?

File: plugins/destination/bigquery/client/client.go [hunk 2]
```diff
@@ -16,6 +16,7 @@ type Client struct {
 	logger     zerolog.Logger
 	spec       specs.Destination
 	metrics    plugins.DestinationMetrics
+	batchSize  int
```
Comment (candiduslynx) [inline plugins/destination/bigquery/client/client.go:19]: I see only initialization of this field.
Where is the propagation to the write?

**Review by candiduslynx — CHANGES_REQUESTED** [review]

File: plugins/destination/bigquery/client/client.go (hunk 2 above)
Comment (bbernays) [reply plugins/destination/bigquery/client/client.go:19]: https://github.com/cloudquery/cloudquery/pull/5681

**Pull request merged**